src/molecules/LiistPlaceCardNew.js

```diff
@@ -210,31 +210,55 @@ export class LiistPlaceCardNew extends LitElement {
     super();
   }
 
+  // after the component is first rendered in its initial state (displayed to user)
+  // this function gets executed
+  firstUpdated() {
+    this.setImage();
+  }
+
+  setImage() {
+    const placeThumbnail = this.shadowRoot.querySelector("img.place-thumbnail");
```

Comment:
`this.shadowRoot.querySelector()` is used if you want to grab HTML elements that you specified in the `render()` function.